doc/sphinx/practical-tools/utilities.rst

```diff
@@ -91,8 +91,10 @@ Such command generates the following files:
 CoqMakefile
   is a generic makefile for ``GNU Make`` that provides
   targets to build the project (both ``.v`` and ``.ml*`` files), to install it
```

Comment:
```suggestion
  a makefile for ``GNU Make`` with
  targets to build the project (both ``.v`` and ``.ml*`` files), install it
```

Reply:
Since the next item has the form of a sentence: "CoqMakefile.conf contains", I would keep the "is", wouldn't I? I would otherwise adopt the light form of the suggestion.

A side question is why is it better to call it `CoqMakefile`, rather than just `Makefile`?

Reply:
> Since the next item has the form of a sentence: "CoqMakefile.conf contains", I would keep the "is", wouldn't I?

It's not necessary to have complete sentences with a list of items, especially when the item names ("CoqMakefile", etc.) are on separate lines.  Nor is it necessary to be fully consistent with "contains".  Suppose you have a list of a dozen items.  It is repetitive and uninteresting to repeat "is" on every single one of them.  (The "Such command" above could be removed for the same reason; you really don't need to see "This tactic ..."  or "This command ..." beginning the description of hundreds of those constructs.)  

Is French stricter about everything being full sentences?

I leave it up to you.

> A side question is why is it better to call it `CoqMakefile`, rather than just `Makefile`?

An excellent point.  It would simplify usage and the doc a little.